JetAnalyzers/python/JetReconstruction_cff.py

```diff
@@ -87,7 +87,9 @@
 #!
 #! PF JETS CHS
 #!
-ak1PFCHSJets    = ak1PFJets.clone( src = 'pfNoPileUpJME' )
+
```

Comment:
Please remove these newlines to keep the style the same among the various sections of the code. You should strive to keep the formatting consistent throughout the code.

Reply:
Okay!